samples/java-security-usage/k8s/deployment.yml

```diff
@@ -34,6 +34,9 @@ spec:
             - name: xsuaa2
```

Comment:
looks strange
name xsuaa2 but mountpath ...xsuaa-instance-1

Reply:
that's updated in #615

Reply:
ok